templates/legal/index.html

```diff
@@ -9,7 +9,7 @@
   <div class="row">
     <div class="col-8">
       <h1>Ubuntu legal terms and&nbsp;policies</h1>
```

Comment:
Not sure if relevant, but this title is different to the copydoc

Reply:
Fixed thanks, good spot.